benchmarks.md

```diff
@@ -3,89 +3,117 @@ layout: base
 
 ### Probabilistic
 
-
-* HRRRE
-
-Calibration of probabilistic benchmark forecasts is a topic of discussion.
-
+* GEFS cloud cover to ranked irradiance ensemble
+* Climatology (not yet implemented)
+* Persistence ensemble (not yet implemented)
+
+### Automated generation
+
+Operational benchmark forecasts are currently produced for every
+reference forecast site in the Arbiter's database. Forecasts are currently only produced
+using the [intraday and longer horizons](#Intraday-and-longer-horizons)
+models described above. For these automated benchmark forecasts, we choose to define
+[forecast attributes](/definitions/#forecastattrs) such that new forecasts
+would be issued at midnight local time for each site. Daylight Saving Time
+is not included when determining the issue time. For example, for the
+Table Mountain SURFRAD site, the following parameters apply (interval label
+is ending and interval length is 1 hour for all):
+
+| Model | Issue time of day &nbsp;&nbsp;&nbsp;| Run length &nbsp;&nbsp;&nbsp;| Lead time to start &nbsp;&nbsp;&nbsp;|
+|-------|:-----------------:|:----------:|:------------------:|
+| GFS day ahead | 7Z | 1 day | 1 day |
+| NAM current day | 6Z | 1 day | 1 hour |
+| HRRR intraday | 0Z | 6 hours | 1 hour |
+| RAP intraday | 0Z | 6 hours | 1 hour |
+
+Please contact the framework administrators if you have suggestions for
+improvements to the configuration of the automatically generated
+benchmarks or if you require a custom benchmark configuration for your
+application (e.g. a forecast trial).
 
 ## User-supplied benchmark forecasts
 
-
 Some evalulation applications may require users to provide their own
 benchmark forecasts. For example, Solar Forecasting 2, Topic Area 2
 teams may choose to run an earlier version of WRF Solar and then upload
-its forecasts for particular evaluation points. Benchmark models should
-be run in consultation with the framework administrators. The framework
-will accept uploaded forecasts for predefined evaluation point or
-area-average values. The framework will not accept gridded datasets.
-
+its forecasts for existing or new evaluation sites. Users may then
+identify the forecast as a benchmark when creating a forecast
+evaluation [report](/documentation/dashboard/#create-new-report) so that
+forecast skill metrics will be calculated with respect to the benchmark.
 
-## PV power forecasts
-
-
-The Arbiter will include functionality for converting benchmark solar
-irradiance forecasts into benchmark solar power forecasts in a simple,
-auditable manner. The Arbiter will use solar power modeling functions
-from the [pvlib-python](https://pvlib-python.readthedocs.io/en/latest/)
-library.
+Users are encouraged to track metadata about the forecast using the
+*extra parameters* field of the forecast creation
+[form](documentation/dashboard/#create-new-forecast). Key modeling details
+such as grid spacing and schemes, or even a whole WRF namelist, may be
```

Comment:
I mean, we don't want people to go overboard... Better would be the version of your version controlled namelist

Reply:
`cat run_wrf.csh`